x/tokenfactory/client/cli/query.go

```diff
@@ -20,7 +20,7 @@ func GetQueryCmd() *cobra.Command {
 	cmd := osmocli.QueryIndexCmd(types.ModuleName)
 
 	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
-	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
+	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomsFromCreator)
```

Comment:
Can you please do two things:

1. Add the GetCmdDenomBeforeSendHook query here as well
2. Test them locally on mainnet and ack that they behave as expected

As soon as thats done, please ping me and I will merge :)

Reply:
Done, please check